dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,151 @@
+package dashboards
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// ResourceLakeviewDashboard manages lakeview dashboards
+func ResourceLakeviewDashboard() common.Resource {
+	s := common.StructToSchema(dashboards.Dashboard{},
+		func(s map[string]*schema.Schema) map[string]*schema.Schema {
+			s["create_time"].Computed = true
+			s["dashboard_id"].Computed = true
+			s["display_name"].Optional = false
+			s["display_name"].Required = true
+			s["etag"].Computed = true
+			s["lifecycle_state"].Computed = true
+			s["parent_path"].Optional = false
```

Comment:
It's more to the corresponding team, but I'm wondering why we still don't allow to change parent path - this already leads to a lot of problems with redash dashboards.

@mgyucht should we ask lakeview team about plans for it?